pwsh_module/pwsh_bolt_internal.ps1

```diff
@@ -68,3 +68,17 @@ function Get-BoltCommandline {
 
   Write-Output $params
 }
+
+function Get-BoltVersion{
+  [CmdletBinding()]
+  param()
+  $IsLinuxEnv = (Get-Variable -Name "IsLinux" -ErrorAction Ignore) -and $IsLinux
+  $IsMacOSEnv = (Get-Variable -Name "IsMacOS" -ErrorAction Ignore) -and $IsMacOS
+  $IsWinEnv = !$IsLinuxEnv -and !$IsMacOSEnv
+  if($IsWinEnv){
```

Comment:
Is checking the module's version not guaranteed to work on macOS/Linux? Importing the module on macOS and running `[string](Get-Module -Name PuppetBolt).Version` seems to work fine at least.

Reply:
The original intent was to provide a faster way to get the version of Bolt without executing ruby. Since the `version.txt` isn't updated reliably, moving to the `PuppetBolt` module version makes sense, so long as we always keep the module the same version as the released bolt version.

Since the module can be released without making a bolt MSI release, theoretically the two could be versioned separately. If we commit to always have a 1:1 release of the MSI and the module, then I think we can safely use the module version.